src/character.cpp

```diff
@@ -3252,8 +3252,18 @@ units::mass Character::bodyweight() const
 
 int Character::height() const
 {
+    int height = 175;
+
+    if( get_size() == MS_SMALL ) {
```

Comment:
i suggest making these a modifier instead of a straight up equals sign. that way when we move on to adding height as a character creation option in a later PR, we don't have to run around trying to figure out why your mouse character doesn't differ in metabolism if you started out taller.